Sources/Spezi/Standard/Module+Standard.swift

```diff
@@ -16,16 +16,17 @@ extension Module {
 
 
 extension Module {
-    /// Defines access to the shared ``Standard`` actor.
+    /// Defines access to the shared `Standard` actor.
+    ///
+    /// A ``Module`` can define the injection of a ``Standard`` using the @``Module/StandardActor`` property wrapper.
+    ///
+    /// - Note: You can access `@StandardActor` once your ``Module/configure()-5pa83`` method is called (e.g., it must not be used in the `init`)
```

Comment:
Might be cool to provide this as a tip in the main Module documentation, I think this could be a common use case.